src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -179,7 +179,7 @@ private string DetermineProjectUnderTestWithNameFilter(string projectUnderTestNa
             var projectReferencesMatchingNameFilter = projectReferences.Where(x => x.Contains(projectUnderTestNameFilter, StringComparison.OrdinalIgnoreCase));
             if (!projectReferencesMatchingNameFilter.Any())
             {
-                stringBuilder.Append("No project reference matched your --project-file=");
+                stringBuilder.Append("No project reference matched your --project=");
```

Comment:
```suggestion
                stringBuilder.Append("No project reference matched the given project filter");
```

I would like to remove all direct references to what options look like from Stryker.Core as it has no relation to what Stryker.CLI decides the commandline/config file option is called.